### DISCUSSION TITLE
Import to new table from csv/xlsx

### DISCUSSION THREAD

**Discussion by KrzysztofDusko** [dbeaver/dbeaver#23149] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Import data with built in creation of table (with detection of data types) will be really useful in some situations, like ad hoc reporting.

**Describe the solution you'd like**
Well, after selecting file should be scanning phase (of part or full file), based on table content, table creation, and insert data to newly created table.

**Describe alternatives you've considered**

**Additional context**

**Comment by ShadelessFox** [comment, marked as answer]
Hello @KrzysztofDusko,

Are you aware of the data transfer feature? It allows you to import data from various sources, including CSV files, into existing and new tables.

You can learn more here: https://github.com/dbeaver/dbeaver/wiki/Data-transfer

**Comment by KrzysztofDusko** [comment]
Thank you!
